Module.php

```diff
@@ -112,6 +112,19 @@ class Module extends \yii\base\Module implements BootstrapInterface
     public $traceLine = self::DEFAULT_IDE_TRACELINE;
 
     /**
+     * @var array the rule which defines who allowed to switch user identity.
+     * Access Control Filter single rule. Ignore: actions, controllers, verbs.
+     * Settable: allow, roles, ips, matchCallback, denyCallback.
+     * By default deny for everyone. Recommendation: can allow for administrator
+     * or developer (if implement) role: ['allow' => true, 'roles' => ['admin']]
+     * @see http://www.yiiframework.com/doc-2.0/guide-security-authorization.html
+     * @since 2.0.10
+     */
+    public $allowedUserSwitch = [
```

Comment:
Foolproof. If allowed, then will broken BC default security authorization. Access without custom settings to the panel shall not allowing user interfere with RBAC.